src/Microsoft.DotNet.Darc/src/DarcLib/Actions/Remote.cs

```diff
@@ -801,7 +801,7 @@ private IEnumerable<DependencyDetail> CalculateLeavesOfCoherencyTrees(IEnumerabl
                 {
                     _logger.LogInformation($"There's a difference between the number of files in the eng/common folder" +
```

Comment:
I'd move this whole block down below the foreach below, and if any files were added or deleted, list out the full set.